scripts/genertate/README.md

```diff
@@ -0,0 +1,33 @@
+
+Some of the examples below may include Unicode text characters. Set the environment variable:
+```bash
+export PYTHONIOENCODING=UTF-8
```

Comment:
Why do we need to specify the PYTHONENCODING?

Reply:
https://github.com/openai/gpt-2/blob/master/DEVELOPERS.md

It is said that some of the examples below may include Unicode text characters.